configuration/pipeline_config.json

```diff
@@ -7,20 +7,7 @@
       "TokenFileURL": "gs://avf-credentials/imaqal-text-it-token.txt",
       "ContactsFileName": "imaqal_covid19_contacts",
       "ActivationFlowNames": [
-        "covid19_som_s01e01_activation",
-        "covid19_som_s01e02_activation",
-        "imaqal_covid19_mag_s01e01_activation",
-        "imaqal_covid19_mag_s01e02_activation",
-        "imaqal_covid19_mag_s01e03_activation",
-        "imaqal_covid19_mag_s01e04_activation",
-        "imaqal_covid19_mag_s01e05_activation",
-        "imaqal_covid19_mag_s01e06_activation",
-        "imaqal_covid19_mag_s01e07_activation",
-        "imaqal_covid19_mag_s01e08_activation",
-        "imaqal_covid19_mag_s01e09_activation",
-        "imaqal_covid19_mag_s01e10_activation",
-        "imaqal_covid19_mag_s01e11_activation",
-        "imaqal_covid19_mag_s01e12_activation"
```

Comment:
Why are we removing all the old shows? Was this accidentally committed from testing?

Reply:
accidentally committed thanks for catching this